openhands/runtime/remote/runtime.py

```diff
@@ -349,16 +354,17 @@ def run_action(self, action: Action) -> Observation:
                 return NullObservation('')
             action_type = action.action  # type: ignore[attr-defined]
             if action_type not in ACTION_TYPE_TO_CLASS:
-                return ErrorObservation(f'Action {action_type} does not exist.')
+                return FatalErrorObservation(
+                    f'[Runtime (ID={self.runtime_id})] Action {action_type} does not exist.'
+                )
```

Comment:
I'm not sure this is fatal, IIRC json-based agents expect to receive this when the LLM sends the wrong json, and we do intently send that back to the LLM.

Reply:
Oh wait, this is in the runtime, ignore that. I thought it was the action creation code. Does the same 😅